triton-vm/src/table/jump_stack_table.rs

```diff
@@ -305,16 +315,12 @@ impl JumpStackTable {
 }
 
 impl ExtJumpStackTable {
-    pub fn with_padded_height(num_trace_randomizers: usize, padded_height: usize) -> Self {
+    pub fn with_padded_height() -> Self {
```

Comment:
Is this function (and similar functions for other tables) used anywhere? It seems strange to have a function name that anticipates an argument and then not supply that argument. Also, I noticed that the same function is defined on `ExtTableCollection` but with the argument there.

Reply:
Good catch. This might be a remnant of some previous refactoring effort. To be frank, I lost track of some of the bits during the refactor for this PR.

Reply:
Fixed by 59e5b32